From 62b8f5c6cf5c2cab7b380ca880fc34e9b2523c75 Mon Sep 17 00:00:00 2001
From: Alf Gaida <agaida@siduction.org>
Date: Fri, 30 Sep 2016 17:19:47 +0200
Subject: [PATCH 1/4] Remove cpack (#49)

* remove "building with cpack" from CMakeLists.txt - not used anymore

* Added very basic .gitattributes
---
 .gitattributes | 17 +++++++++++++++++
 CMakeLists.txt | 10 ----------
 2 files changed, 17 insertions(+), 10 deletions(-)
 create mode 100644 .gitattributes

diff --git a/.gitattributes b/.gitattributes
new file mode 100644
index 0000000..0966604
--- /dev/null
+++ b/.gitattributes
@@ -0,0 +1,17 @@
+# remove files from deployment using `git archive`
+
+# git files
+.gitattributes export-ignore
+.gitignore     export-ignore
+
+
+# several files and directories we never want to export 
+# a little bit belt and braces as the most of these files
+# should never ever be in the repository
+
+.*~        export-ignore
+.kdev4     export-ignore
+
+/build export-ignore
+/temp  export-ignore
+/tmp   export-ignore
diff --git a/CMakeLists.txt b/CMakeLists.txt
index b569dfd..ebe033c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -148,13 +148,3 @@ install(TARGETS
     RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
     COMPONENT Runtime
 )
-
-# building tarball with CPack -------------------------------------------------
-include(InstallRequiredSystemLibraries)
-set(CPACK_PACKAGE_VERSION_MAJOR ${LXQT_MAJOR_VERSION})
-set(CPACK_PACKAGE_VERSION_MINOR ${LXQT_MINOR_VERSION})
-set(CPACK_PACKAGE_VERSION_PATCH ${LXQT_PATCH_VERSION})
-set(CPACK_GENERATOR TBZ2)
-set(CPACK_SOURCE_GENERATOR TBZ2)
-set(CPACK_SOURCE_IGNORE_FILES /build/;.gitignore;.*~;.git;.kdev4;temp)
-include(CPack)
-- 
2.8.3

